domain-services/ifix-master-data-service/README.md

```diff
@@ -1,5 +1,16 @@
 # iFIX-Master-Data-Service
 
+## Connect to MongoDB pod
```

Comment:
"Connect to MongoDB through Playground pod"

Reply:
done